README.md

```diff
@@ -1,5 +1,10 @@
 # HKN Server
 
+## Deploys
+
+- `Production`: http://api.hknucsd.com/
```

Comment:
Is endpoint protection on yet?

Reply:
Endpoint protection?

Reply:
@godwinpang what endpoint protection are you talking about?

Reply:
uh auth; just wondering if we'll wake up and see 2k events in our db

Reply:
ohh uhhh not yet
I wasn't sure which auth middleware to put for each endpoint : p

Reply:
ah ok let's not dos ourselves - can you put officerauth on the endpoints that aren't GET?

Reply:
for sure, on this PR or another?

Reply:
another

Reply:
Oh and what about AuthController and OldUserController? What should I do with those?

Reply:
And also what about event sign in and rsvp endpoints? Aren't those supposed to be public?

Reply:
ya event signin and rsvp endpoints are fine - we can leave those open

Reply:
actually, what happens when someone signs up for a nonexistent event?

Reply:
we can comment out the app.use(AuthRouter) and app.use(OldUserRouter) for now

Reply:
For sure